packages/hub/src/lib/parse-safetensors-metadata.ts

```diff
@@ -8,8 +8,10 @@ import { downloadFile } from "./download-file";
 import { fileExists } from "./file-exists";
 import { promisesQueue } from "../utils/promisesQueue";
 
-const SINGLE_FILE = "model.safetensors";
-const INDEX_FILE = "model.safetensors.index.json";
+export const SINGLE_FILE = "model.safetensors";
+export const INDEX_FILE = "model.safetensors.index.json";
```

Comment:
exporting them so that we can use it in moon

Reply:
let's rename them if we export them

```suggestion
export const SAFETENSORS_DEFAULT_FILE = "model.safetensors";
export const SAFETENSORS_DEFAULT_INDEX_FILE = "model.safetensors.index.json";
```

Reply:
wdyt b94af07eecc631b499d99dc87c68ce3f02090b33